src/v2/cookbook/how-to-use-the-vue-plugin-api.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Making use of the Vue Plugin API
+type: cookbook
+order: 15
+---
+
+## Base Example
+
+Vue exposes a really nifty API for creating plugins, plugins allow us to modify and add features to Vue. This cookbook is aimed at showing you how to take a simple idea and turn it into a full-fledged Vue plugin.
```

Comment:
Nice intro :)

The idea could be carried through the sentence to be a little more clear: 
Vue exposes a really nifty API for creating plugins, which allow us to modify and add features.